models/issue_comment.go

```diff
@@ -568,47 +574,68 @@ func GetCommentByID(id int64) (*Comment, error) {
 	return c, nil
 }
 
-func getCommentsByIssueIDSince(e Engine, issueID, since int64) ([]*Comment, error) {
-	comments := make([]*Comment, 0, 10)
-	sess := e.
-		Where("issue_id = ?", issueID).
-		Where("type = ?", CommentTypeComment).
-		Asc("created_unix")
-	if since > 0 {
-		sess.And("updated_unix >= ?", since)
+// FindCommentsOptions describes the condtions to Find comments
+type FindCommentsOptions struct {
+	RepoID  int64
+	IssueID int64
+	Since   int64
+	Type    CommentType
+}
+
+func (opts *FindCommentsOptions) toConds() builder.Cond {
+	var cond = builder.NewCond()
+	if opts.RepoID > 0 {
+		cond = cond.And(builder.Eq{"issue.repo_id": opts.RepoID})
+	}
+	if opts.IssueID > 0 {
+		cond = cond.And(builder.Eq{"issue.id": opts.IssueID})
```

Comment:
Could use `"comment.issue_id"` instead of `"issue.id"`, so that we can avoid joining if `opts.RepoID` is 0.